leetcode/0001/TwoSum.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections.Generic;
+
+public class Solution
+{
+    public int[] TwoSum(int[] nums, int target)
+    {
+        return SolveWithOnePass(nums, target);
+    }
+
+    public int[] SolveWithOnePass(int[] nums, int target)
```

Comment:
I didn't like that my solution was only in the ~85% range.  Studying what LeetCode provided, I understood their one-pass approach.  Taking that idea and implementing got the solution to be faster, but still not nearly as close as I wanted.  Then I realized that I wasn't initializing the capacity of the dictionary.  Upon doing that, the solution was in the 99% range.